checkout-service/test/react/react.test.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import ReactDOM from 'react-dom';
+import { render, unmountComponentAtNode } from "react-dom";
+import { act } from "react-dom/test-utils";
+
+import Enzyme, { shallow, configure, mount } from "enzyme";
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
+configure({ adapter: new Adapter() });
+
+import Checkout from '../../client/src/components/Checkout.jsx';
+import { PriceDeliveryAndStock } from '../../client/src/components/CheckoutDetails.jsx';
+
+describe("Checkout component", () => {
+  test("renders", () => {
+    const wrapper = shallow(<Checkout />);
+    expect(wrapper.exists()).toBe(true);
+  });
+});
+
+describe("Checkout PriceDeliveryAndStock component", () => {
+  test("renders", () => {
+    const wrapper = shallow(<PriceDeliveryAndStock />);
+    expect(wrapper.exists()).toBe(true);
+  });
+
+  test("renders passed price prop", () => {
+    const wrapper = mount(<PriceDeliveryAndStock />);
```

Comment:
Just a question. Why did you pick mount here and shallow above? Will shallow work on both places?

Reply:
For some reason my test wouldn't pass using shallow. According to what I read, mount goes deeper into the children of the component. However, that component has no children. I feel like it might have something to do with the use of styled components or passing props maybe.